modules/rubiconBidAdapter.js

```diff
@@ -93,150 +94,238 @@ export const spec = {
       return false;
     }
 
-    if (bid.mediaType === 'video') {
-      if (typeof params.video !== 'object' || !params.video.size_id) {
+    if (spec.hasVideoMediaType(bid)) {
+      // support instream only
+      if (utils.deepAccess(bid, `mediaTypes.${VIDEO}`) && utils.deepAccess(bid, `mediaTypes.${VIDEO}.context`) !== 'instream' ||
+        typeof params.video !== 'object' || !params.video.size_id) {
```

Comment:
I think this may break the backwards compatibility, when we use `bidRequest.mediaType === VIDEO`, `utils.deepAccess(bid, `mediaTypes.${VIDEO}`) && utils.deepAccess(bid, `mediaTypes.${VIDEO}.context`) !== 'instream' || typeof params.video !== 'object' || !params.video.size_id` can return **false** but `utils.deepAccess(bid, `mediaTypes.${VIDEO}.context`) !== 'instream' || !utils.deepAccess(params, 'video.size_id')` will always return **true**

Reply:
@idettman Sorry my comments is not clear, the original code `utils.deepAccess(bid, 'mediaTypes.${VIDEO}') && utils.deepAccess(bid, 'mediaTypes.${VIDEO}.context') !== 'instream' || typeof params.video !== 'object' || !params.video.size_id` should be the correct one.

Reply:
Thanks for clarifying, I should have read more carefully. Fix committed